beacon-chain/node/node.go

```diff
@@ -97,6 +97,12 @@ func NewBeaconNode(cliCtx *cli.Context) (*BeaconNode, error) {
 		params.LoadChainConfigFile(chainConfigFileName)
 	}
 
+	if cliCtx.IsSet(flags.SlotsPerArchivedPoint.Name) {
+		c := params.MainnetConfig()
```

Comment:
Why do you use `MainnetConfig` here? Wouldn't this break that flag for minimal config runs?

Reply:
Nice catch. I meant to use BeaconConfig